operator/internal/controller/vectorized/cluster_controller_configuration_drift_test.go

```diff
@@ -17,6 +17,28 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestDiffWithNull(t *testing.T) {
```

Comment:
test is expected to fail, currently it's a diff,keeping operator spinning forever.

Reply:
shall succeed now with the fix commit